src/idiom275ex.fz

```diff
@@ -0,0 +1,13 @@
+ex275 is
+
+  s := """
+       00000000\
+       11111111\
+       01111111\
+       00000001\
+       00101010
+       """
+
+  say (array s.byte_length/8 i->
```

Comment:
Good, another option would something like `s.as_codepoints.chunk(8).map x->x.parse_integer u8 2`

Reply:
I had to add an extra `foldf`, but I still think this approach is better because it doesn't iterate over the string multiple times. Also I think it is not less understandable than the previous version.